src/staging/src/components/park/parkDates.js

```diff
@@ -1,18 +1,39 @@
-import React from "react"
+import React, { useState } from "react"
 import moment from "moment"
 
-import Heading from "./heading" // TODO this is MUI, but in many places
 import HTMLArea from "../HTMLArea"
+import Row from "react-bootstrap/Row"
+import Col from "react-bootstrap/Col"
+import Accordion from "react-bootstrap/Accordion"
+import Container from "react-bootstrap/Container"
+import Heading from "./heading"
+import HtmlContent from "./htmlContent"
+import StaticIcon from "./staticIcon"
+import { ParkAccessFromAdvisories } from "../../components/park/parkAccessStatus"
 
 export default function ParkDates({ data }) {
   const parkOperation = data.parkOperation || {}
   const subAreas = data.subAreas || []
+  subAreas.sort((a, b) => (a.parkSubArea >= b.parkSubArea ? 1 : -1))
```

Comment:
I think it's fine for now, just an FYI for future: there's a lot of prop mutation happening in this component. `data.subAreas` gets sorted in place, each subarea gets a few properties added, etc. These changes will propagate up to the parent component. It's not a big issue because this component is only used once on the page, but it violates the principle of read only props:
https://reactjs.org/docs/components-and-props.html#props-are-read-only

Reply:
Updated.